nixpkgs_merge_bot/nix.py

```diff
@@ -128,5 +128,9 @@ def merge_check(
         logging.info(
             "merge is permitted, no decline reasons, but we are running in dry-run mode"
         )
+    else:
+        decline_reasons.append(
+            "bot is running in dry-run mode"
+        )
```

Comment:
I don't think the code even reaches this part though, since you `return` early above for the restricted authors check